sdk-resources/resources/api.mustache

```diff
@@ -8,7 +8,7 @@
 
 .DESCRIPTION
 
-{{{description}}}{{^description}}No description available.{{/description}}
+{{{description}}}{{^description}}{{{notes}}}{{^notes}}No description or notes available.{{/notes}}{{/description}}
```

Comment:
Would it make sense to add something here? Is description pulled from another openAPI field that we should be utilizing?

Reply:
No I don't believe so. When you define the endpoint like so it will be read in as a description.

```yaml
get:
  operationId: getAccessProfile
  tags:
    - Access Profiles
  summary: Get an Access Profile
  description: This API returns an Access Profile by its ID.
```

When we use the `>-` notation, this is what converts it into a note. Therefore you could only have one or the other.
```yaml
get:
  operationId: getAccessProfile
  tags:
    - Access Profiles
  summary: Get an Access Profile
  description: >-
    This API returns an Access Profile by its ID.


    A token with API, ORG_ADMIN, ROLE_ADMIN, ROLE_SUBADMIN, SOURCE_ADMIN, or SOURCE_SUBADMIN authority is required to
    call this API.
```